src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -15,7 +15,20 @@
 
 - name: Install nginx
   become: yes
-  apt: name="nginx=1.17.3-1~bionic" state=present update_cache=yes cache_valid_time=3600
+  apt: name="{{ nginx_apt_package }}" state=present update_cache=yes cache_valid_time=3600
+  register: install_nginx
+  failed_when: no
+  notify: 'quick start nginx'
+
+- name: Fix broken install
+  become: yes
+  shell: 'apt --fix-broken install -y'
+  when: "install_nginx is defined and '--fix-broken' in install_nginx.stderr"
```

Comment:
do you know why this was "broken"? It seems strange to automate fixing broken dependencies by default without alerting the user first

Reply:
Hmm. Yeah, that makes sense. I noticed this was a part of the process when I did it manually on staging and that it worked fine, but that's probably not a great response. I can try and understand this better before we merge, or at least paste the output here for others to review